interfaces/screen-capture.idl

```diff
@@ -0,0 +1,27 @@
+// GENERATED CONTENT - DO NOT EDIT
+// Content of this file was automatically extracted from the
+// "Screen Capture" spec.
+// See: https://w3c.github.io/mediacapture-screen-share/
+
+partial interface Navigator {
+    Promise<MediaStream> getDisplayMedia(optional MediaStreamConstraints constraints);
+};
+
+partial dictionary MediaTrackConstraintSet {
+             ConstrainDOMString displaySurface;
+             ConstrainBoolean logicalSurface;
+             ConstrainDOMString cursor;
+};
+
+enum DisplayCaptureSurfaceType {
```

Comment:
Yes, it's a bid odd that these enums aren't used. They're referenced in prose, similar to what you can see if searching for "VideoFacingModeEnum" in https://w3c.github.io/mediacapture-main/#dom-videofacingmodeenum

Reply:
Yep. It's harmless since defining an IDL enum doesn't do anything unless used as an attribute/argument type somewhere, but could come up as noise if we in the future build tooling to detect IDL "dead code". I looked at the spec and think that using prose do define the equivalent behavior would be the fix if later needed.